app/services/cycle_timetable.rb

```diff
@@ -129,13 +133,15 @@ def self.apply_2_deadline_second_reminder
   end
 
   def self.between_cycles_apply_1?
-    Time.zone.now > apply_1_deadline &&
-      Time.zone.now < apply_reopens
+    (Time.zone.now > apply_1_deadline &&
+      Time.zone.now < apply_reopens) ||
+      Time.zone.now < apply_opens
```

Comment:
I think this makes sense but will it lead to confusing messages from `CandidateInterface::ReopenBannerComponent` (because this method is used to determine whether the banner is displayed I think)? 
https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/components/candidate_interface/reopen_banner_component.html.erb

Reply:
The reopen banner should continue to show up until apply opens. So i think it should continue to work as normal right?

Reply:
Yes should appear I guess but the dates might be wrong as it seems to assume that the current cycle year is the last year. To be honest it's a bit confusing so maybe I'm missing something and it's all fine.

Reply:
I've tested this locally and it's all good. The `ReopenBannerComponent` handles the switch from one cycle to the next in https://github.com/DFE-Digital/apply-for-teacher-training/blob/main/app/components/candidate_interface/reopen_banner_component.rb#L47-L51 so as far as I can see it always shows the correct dates.